### PR TITLE
Bug 1899587: Fix RGW Total Used Query

### PR DESCRIPTION
**Pull request by bipuladh**
Stop subtracting NooBaa usage from RGW usage

### CODE REVIEW

**Comment by kyoto** [comment]
FYI @openshift/openshift-team-monitoring

**Review by simonpasquier — COMMENTED** [review]
/lgtm

**Comment by bipuladh** [comment]
/hold
need ack from Storage team before we can get this merged.

File: frontend/packages/noobaa-storage-plugin/src/queries.ts
Comment (anmolsachan) [inline frontend/packages/noobaa-storage-plugin/src/queries.ts:63]:
```suggestion
    'sum(NooBaa_providers_physical_size)',
```

File: frontend/packages/noobaa-storage-plugin/src/queries.ts
Comment (anmolsachan) [inline frontend/packages/noobaa-storage-plugin/src/queries.ts:70]:
```suggestion
      'sum(ceph_pool_metadata{name=~"<%= name %>rgw.*"} *on (job, namesapce, pool_id) group_right(name) ceph_pool_stored_raw) + sum(NooBaa_providers_physical_size{type!="S3_COMPATIBLE"})',
```

**Review by anmolsachan — CHANGES_REQUESTED** [review]

File: frontend/packages/noobaa-storage-plugin/src/queries.ts
Comment (anmolsachan) [inline frontend/packages/noobaa-storage-plugin/src/queries.ts:65]:
```suggestion
      'sum(ceph_pool_metadata{name=~"<%= name %>rgw.buckets.data"} *on (job, namesapce, pool_id) group_right(name) ceph_pool_stored) - max(NooBaa_providers_physical_size{type="S3_COMPATIBLE"} or vector(0))',
```

File: frontend/packages/noobaa-storage-plugin/src/queries.ts
Comment (anmolsachan) [inline frontend/packages/noobaa-storage-plugin/src/queries.ts:69]:
```suggestion
      'sum(ceph_pool_metadata{name=~"<%= name %>rgw.buckets.data"} *on (job, namesapce, pool_id) group_right(name) ceph_pool_stored) + max(sum(NooBaa_providers_physical_size{type!="S3_COMPATIBLE"}) or vector(0))',
```

**Review by anmolsachan — CHANGES_REQUESTED** [review]

**Pull request merged**
